index.html

```diff
@@ -296,9 +296,34 @@ <h3>Skills</h3>
             </div>
             </div>
             </section>
-        
-    
+            <section id="contact-form">
+                          
+                   <form id="form-1" action="https://formspree.io/f/form-1" method="post">
```

Comment:
- [ ] Form is not submitting correctly, kindly fix it
![image](https://user-images.githubusercontent.com/62705947/171383219-2bb9aab7-81d4-4d71-af97-e39ca4207360.png)

Reply:
Done as advised